pkg/kritis/metadata/metadata_test.go

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/attestation"
+	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/common"
+	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/grafeas"
+	pkg "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/package"
+	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/vulnerability"
+)
+
+func TestGetVulnerabilityFromOccurence(t *testing.T) {
```

Comment:
Thanks for adding this test too!

Reply:
Actually this is a test that already existed! But I moved it here because I thought it belonged with `TestGetRawAttestationsFromOccurrence`